libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -57,6 +57,22 @@ bool StaticAnalyzer::visit(FunctionDefinition const& _function)
 	solAssert(m_localVarUseCount.empty(), "");
 	m_nonPayablePublic = _function.isPublic() && !_function.isPayable();
 	m_constructor = _function.isConstructor();
+
+	if (m_currentContract->sourceUnit().annotation().experimentalFeatures.count(ExperimentalFeature::V050))
+	{
+		if (_function.isOverride() && !_function.annotation().superFunction)
+			m_errorReporter.declarationError(
+				_function.location(),
+				"Function signature does not override any function defined in parent contracts."
+			);
+
+		else if (!_function.isOverride() && _function.annotation().superFunction)
+			m_errorReporter.declarationError(
+				_function.location(),
+				"Function overrides a parent function but is missing an override specifier."
```

Comment:
Perhaps just "overrides a function"?

Reply:
Can we find out the name of the contract the overridden function is defined in? Perhaps we should even specify a secondary source location.